parlai/mturk/tasks/turn_annotations/html/Person1_index.html

```diff
@@ -0,0 +1,326 @@
+{% extends "core.html" %}
+
+{% block left_pane %}
+<div id="left-pane" class="col-xs-4" style="height: {{frame_height}}px; background-color: #dff0d8;">
+  <h1>Live Chat</h1>
+  <hr style="border-top: 1px solid #555" />
+  <span id="left-pane-text" style="font-size: 14px">
+  </span>
+</div>
+{% endblock %}
+{% block right_pane %}
+<div id="right-pane" style="min-height: 100%; display: flex; flex-direction: column; justify-content: space-between;">
+  <div id="right-top-pane"
+    style="width: 100%; height: 570px; padding-top: 60px; padding-left: 20px; padding-right: 20px; padding-bottom: 20px; overflow:scroll; ">
+    <div id="message_thread" style="width: 100%">
+    </div>
+    <div id="waiting-for-message" class="row" style="margin-left: 0; margin-right: 0; display: none">
+      <div class="alert alert-warning" role="alert" style="float: left; display:table; background-color: #fff">
+        <div id="hourglass" style="margin-top: -1px; margin-right: 5px; display: inline; float: left;">
+          <?xml version="1.0" encoding="utf-8"?><svg width='25px' height='25px' xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 100 100" preserveAspectRatio="xMidYMid" class="uil-hourglass">
+            <rect x="0" y="0" width="100" height="100" fill="none" class="bk"></rect>
+            <g>
+              <path fill="none" stroke="#007282" stroke-width="5" stroke-miterlimit="10"
+                d="M58.4,51.7c-0.9-0.9-1.4-2-1.4-2.3s0.5-0.4,1.4-1.4 C70.8,43.8,79.8,30.5,80,15.5H70H30H20c0.2,15,9.2,28.1,21.6,32.3c0.9,0.9,1.4,1.2,1.4,1.5s-0.5,1.6-1.4,2.5 C29.2,56.1,20.2,69.5,20,85.5h10h40h10C79.8,69.5,70.8,55.9,58.4,51.7z"
```

Comment:
lol what is this?

Reply:
Some terrible hourglass animation that is from the legacy task this started from. I'd say leave it - a Mephisto version of this with React wouldn't have this HTML anyway.